Admin/MonitorExchangeAuthCertificate/ConfigurationAction/Register-AuthCertificateRenewalTask.ps1

```diff
@@ -58,7 +60,17 @@ function Register-AuthCertificateRenewalTask {
         $newScheduledTaskParams = @{
             Execute          = "powershell.exe"
             WorkingDirectory = "$($WorkingDirectory)"
-            Argument         = "-NonInteractive -NoLogo -NoProfile -Command `".\$($ScriptName) -ValidateAndRenewAuthCertificate `$true -IgnoreUnreachableServers `$$($IgnoreOfflineServers) -IgnoreHybridConfig `$$($IgnoreHybridConfig) -Confirm:`$false`""
+        }
+
+        $basicArgumentParameters = "-NonInteractive -NoLogo -NoProfile -Command `".\$($ScriptName) -ValidateAndRenewAuthCertificate `$true -IgnoreUnreachableServers `$$($IgnoreOfflineServers) -IgnoreHybridConfig `$$($IgnoreHybridConfig)"
+        if (-not([System.String]::IsNullOrEmpty($SendEmailNotificationTo))) {
```

Comment:
$SendEmailNotificationTo is [string[]] but this check pretends it's [string]. This will not work consistently. For example, if you provide an actual [string[]] where all the values are empty:

![image](https://user-images.githubusercontent.com/4518572/218325666-f38777ce-a2fe-4306-aae2-3305f9bb38ba.png)

Reply:
Added `ValidatePattern` and switched the check to a `$null` check. Should be sufficient in this case.